src/main/java/creativei/dao/InquiryCustomDao.java

```diff
@@ -20,4 +22,10 @@
     List<Inquiry> findAll();
 
     Inquiry findOne(Long aLong);
+
```

Comment:
Also, no need to add a different method for each request. Club monthly and daily request data respectively

Reply:
monthly weekly and daily request are clubbed together the other two is for count for enrolled in a month and count of inquiry which have case index 3 or case index 4

Reply:
Case index should be passed as an array of all the values and then fetch for all of them

Reply:
ok. The last two can not be clubbed together is that okay?

Reply:
The other 2 can also be subsets of monthly data? If not can be separate requests

Reply:
The three monthly,weekly and daily request have same query on database layer. Service layer have different method for each request.

Reply:
case index passed as array 6fa042b